packages/@okta/i18n/src/properties/login_zh_CN.properties

```diff
@@ -182,7 +182,7 @@ errors.E0000110 = \u60A8\u6240\u8BBF\u95EE\u7684\u94FE\u63A5\u5DF2\u8FC7\u671F\u
 errors.E0000111 = \u65E0\u6CD5\u4FEE\u6539 {0} \u5BF9\u8C61\uFF0C\u56E0\u4E3A\u5B83\u662F\u53EA\u8BFB\u7684\u3002
 errors.E0000112 = \u65E0\u6CD5\u66F4\u65B0\u6B64\u7528\u6237\uFF0C\u56E0\u4E3A\u4ED6\u4EEC\u4ECD\u5904\u4E8E\u6FC0\u6D3B\u8FC7\u7A0B\u4E2D\u3002\u8BF7\u5728\u51E0\u5206\u949F\u540E\u91CD\u8BD5\u3002
 # {0} is a server-side error response, oftentimes coming directly from a vendor.
-errors.E0000113 = {0}\u3002
+errors.E0000113 = {0}.
```

Comment:
@evelynteo-okta - This seems like an invalid change, as it is the only place we're using a period over the full-stop (`\u3002`).

Reply:
@jmelberg-okta You are right. @carloslima-okta  This is a result of dangling separator. It's considered a non-translatable element. We'll have to run QA each time to make sure this doesn't get reverted. I'll fix it in the file. It affects all the East Asian Languages (zh-CN, zh-TW, ja-JP) and Thai (no period  or comma punctuation exist in Thai language). 

The implication for the developers here is that it will throw the product on a spin if there is mismatch between this dangling punctuation in this file that doesn't match the dangling punctuation in another resource bundle.